File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.java
class Solution {
    public String solution(String phone_number) {
        String answer = "";
        
        int length = phone_number.length()-4;
        for(int i=0; i<phone_number.length(); i++){
            if (i<length) answer+="*";
            else answer+=phone_number.charAt(i);
        }
        return answer;
    }
}

File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.swift
func solution(_ phone_number:String) -> String {
    var answer = ""
    var length = phone_number.count - 4
    for (index, i) in phone_number.enumerated() {
        if index < length {
            answer.append("*")
        } else {
            answer.append(i)
        }
    }
    return answer
}